lib/ansible/modules/apt_key.py

```diff
@@ -323,10 +325,10 @@ def import_key(module, keyring, keyserver, key_id):
         # Out of retries
         if rc == 2 and 'not found on keyserver' in out:
             msg = 'Key %s not found on keyserver %s' % (key_id, keyserver)
-            module.fail_json(cmd=cmd, msg=msg)
+            module.fail_json(cmd=cmd, msg=msg, environ_update=lang_env)
```

Comment:
`environ_update` doesn't need to be passed in here.

Reply:
The `module.fail_json` returns an error message. Since the `gpg` is called with `environ_update=lang_env`, then it is a relevant information in case of the error, so the user would be aware what kind of environment update was passed to `gpg` that caused an error.

Please consider accepting passing this parameter here.

Reply:
a 'forced_environment' might be a better name than the internal environ_update if the point is to inform the user

Reply:
The keys are used for informational purposes to the user. I agree that using a more human readable key name would be better rather than using the internal parameter name.

Reply:
@bcoca @samdoran Thank you, I have replaced it to the 'forced_environment'

Reply:
Could you please review it again?